CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.0.0-beta2 (unreleased)
+
+* [Breaking] Remove `ulog` and expose `*zap.Logger` directly.
+* [Breaking] Upgrade `zap` to `v1.0.0-rc.2` (now go.uber.org/zap, was
+    github.com/uber-go/zap)
```

Comment:
Gotcha, makes sense. I was thinking of the readme in the ulog package - it's now quite outdated.